### DISCUSSION TITLE
Unique namespace stdgraph instead of std::graph

### DISCUSSION THREAD

**Discussion by mathisloge** [stdgraph/graph-v2#105] [Ideas]
It would be great if the sources would use an own namespace like `stdgraph` instead of `std::graph`, similar to https://github.com/NVIDIA/stdexec .

Later if the graph library is added to the standard, it would make it more maintainable (from a user perspective) to use either the graph-v2 library or the vendor provided stl implementation.

**Comment by pratzl** [comment]
I agree. Using std::graph is a bit presumptuous, as there's no guarantee it will get into the standard.
The namespace used now is just "graph".
